prog/dftb+/lib_dftbplus/parser.F90

```diff
@@ -135,7 +135,7 @@ subroutine parseHsdTree(hsdTree, input, parserFlags)
     type(fnode), pointer :: hsdTree
 
     !> Returns initialised input variables on exit
-    type(TInputData), intent(out) :: input
+    type(TInputData), intent(inout) :: input
```

Comment:
What is the reason for requiring (inout) instead of (out)?

Reply:
Must have been a remainder of some intermediate state. Corrected.